src/style/globalStyles.ts

```diff
@@ -22,9 +22,9 @@ import { isElectron } from '../utils/misc'
 import resets from './resets'
 import tags from './tags'
 
-export const appHeaderHeight = '50px'
+export const appHeaderHeight = '60px'
```

Comment:
We could name it `appHeaderHeightPx` and use a number.
Same with `walletSidebarWidthPx`

Reply:
done